_posts/2019-11-07-snowplow-javascript-tracker-2.12.0-released/index.md

```diff
@@ -41,7 +41,9 @@ In this release we also focused on ensuring our use of Local Storage is robust a
 
 2. We also fixed an issue in the local storage queue that the Snowplow JavaScript Tracker utilises to cache events before they are sent to the collector in a batch or to be queued if the event fails to send (for example, due to connectivity issues). Before this release, there was no upper limit on the size of the queue and this could impact the performance of the web application if it also relies on Local Storage. Most browsers only allow 5MB of Local Storage per site, so allowing control of limiting the Snowplow JavaScript Tracker use of local storage allows web developers to better decide how they wish to utilise their Local Storage limits (Github [#764][764]).
 
-By default this new limit is set to 1000 events in the queue, but it can be configured with the `maxLocalStorageQueueSize` argument when initialising the tracker.
+    NB. In the event the queue becomes full (due to connectivity issues, ad blockers or collector outages) events will be dropped. This is a change in behvaiour from previous releases, which would have only dropped events when Local Storage was full for a site, however we expect with the default value of 1000 that dropped events are unlikely to occur.
```

Comment:
behvaiour -> behaviour

Not sure about the function in the case of ad blockers - surely never sent? I'd scope down to "connectivity issues", which covers collector outages too.

Reply:
The ad blocker will block them so they end up in the queue. This is really the main issue that we were fixing with this bug, with ad blockers on the queue just keeps filling up.
I've fixed the typo and altered to connectivity issues and ad blockers.